recipes/libyuv/all/patches/001-cmake.patch

```diff
@@ -0,0 +1,55 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -6,6 +6,9 @@
+ CMAKE_MINIMUM_REQUIRED( VERSION 2.8 )
+ OPTION( TEST "Built unit tests" OFF )
+ 
++include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
++conan_basic_setup()
```

Comment:
#3141 do we prefer a `cmake_wrapper` or is either method fine?

Reply:
It is better to use `cmake_wrapper`.